tests/algo/res/assembling.xml

```diff
@@ -0,0 +1,100 @@
+  </macroConnection>
+  <singleAssociation id="MESURE_MODELE_1_VL4">
+    <bus voltageLevel="VL" name="1"/>
+    <bus voltageLevel="VL" name="2"/>
+    <busbarSection id="VL_1"/>
+    <busbarSection id="VL_2"/>
+  </singleAssociation>
+  <multipleAssociation id="SHUNTS_MODELE_1_VL4">
+    <shunt voltageLevel="VL"/>
+  </multipleAssociation>
+  <dynamicAutomaton id="MODELE_1_VL4" lib="libdummyLib" type="VoltageControlShunt" access="T0|TFIN">
+    <macroConnect macroConnection="ToUMeasurement" id="MESURE_MODELE_1_VL4"/>
+    <macroConnect macroConnection="ToControlledShunts" id="SHUNTS_MODELE_1_VL4"/>
+  </dynamicAutomaton>
+  <singleAssociation id="MESURE_MODELE_1_TEST2">
+    <bus voltageLevel="VL" name="2.2"/>
+    <busbarSection id="VL_2"/>
+  </singleAssociation>
+  <multipleAssociation id="SHUNTS_MODELE_1_TEST2">
+    <shunt voltageLevel="VL"/>
+  </multipleAssociation>
+  <dynamicAutomaton id="MODELE_1_TEST2" lib="libdummyLib" type="VoltageControlShunt" access="T0|TFIN">
+    <macroConnect macroConnection="ToUMeasurement" id="MESURE_MODELE_1_TEST2"/>
+    <macroConnect macroConnection="ToControlledShunts" id="SHUNTS_MODELE_1_TEST2"/>
+  </dynamicAutomaton>
+  <singleAssociation id="MESURE_MODELE_1_TEST3">
+    <bus voltageLevel="VL" name="3A1"/>
+    <busbarSection id="VL_2"/>
+  </singleAssociation>
+  <multipleAssociation id="SHUNTS_MODELE_1_TEST3">
+    <shunt voltageLevel="VL"/>
+  </multipleAssociation>
+  <dynamicAutomaton id="MODELE_1_TEST3" lib="libdummyLib" type="VoltageControlShunt" access="T0|TFIN">
+    <macroConnect macroConnection="ToUMeasurement" id="MESURE_MODELE_1_TEST3"/>
+    <macroConnect macroConnection="ToControlledShunts" id="SHUNTS_MODELE_1_TEST3"/>
+  </dynamicAutomaton>
+  <multipleAssociation id="SHUNTS_MODELE_1_VL6">
+    <shunt voltageLevel="VL6"/>
+  </multipleAssociation>
+  <dynamicAutomaton id="DM_SALON" lib="libdummyLib" type="CurrentLimitAutomaton" access="T0|TFIN">
+    <macroConnect macroConnection="CLAToIMeasurement" id="MESURE_I_SALON"/>
+    <macroConnect macroConnection="CLAToControlledLineState" id="ORDER_SALON"/>
+    <macroConnect macroConnection="CLAToAutomatonActivated" id="AUTOMATON_EXISTS_SALON"/>
+  </dynamicAutomaton>
+  <singleAssociation id="MESURE_I_SALON">
+    <line name="0"/>
+  </singleAssociation>
+  <singleAssociation id="ORDER_SALON">
+    <line name="1"/>
+  </singleAssociation>
+  <singleAssociation id="AUTOMATON_EXISTS_SALON">
+    <line name="1"/>
+  </singleAssociation>
+  <singleAssociation id="MESURE_I_VL661">
+    <tfo name="TFO1"/>
+  </singleAssociation>
+  <singleAssociation id="TAP_VL661">
+    <tfo name="TFO1"/>
+  </singleAssociation>
+  <singleAssociation id="AUTOMATON_EXISTS_VL661">
+    <tfo name="TFO1"/>
+  </singleAssociation>
+  <dynamicAutomaton id="DM_VL661" lib="libdummyLib" type="PhaseShifterI" access="T0|TFIN">
+    <macroConnect macroConnection="PhaseShifterToIMeasurement" id="MESURE_I_VL661"/>
```

Comment:
PhaseShifter=>Mesure_I? (same idea for others)

Reply:
What is the issue ?

Reply:
just  that the macroConnection keeps the model name in it whereas it was removed from the model